### PR TITLE
09_tabular: wrong splitting of training/validation set (fix issue #325)

### LINKED ISSUE #325: 09_tabular: wrong splitting of training/validation set
The validation set of the bulldozer dataset is supposed to contain the six-month period from November 2011 to April 2012.
However, the split condition in the book does not do this:
`cond = (df.saleYear<2011) | (df.saleMonth<10)`
This moves only the last three months of 2011 into the validation set, and January to April 2012 are used as training data. This leads to a unfairly improved performance for both the random forest and the neural network!
This has been covered on the forums, but I think it has not been corrected yet in the book:
[https://forums.fast.ai/t/lesson-7-official-topic/69896/269](url)
[https://forums.fast.ai/t/lesson-7-official-topic/69896/273](url)

### PR DESCRIPTION
**Pull request by mariano22**
fix #325 (https://github.com/fastai/fastbook/issues/325)

### CODE REVIEW

**Comment by JohannesStutz** [comment]
Hi, thank you for fixing the split. The problem however is, that the results with the new validation set are very different, and this affects the remaining chapter. Including the conclusion that a neural network performs great, or that this result is better than any score shown on the Kaggle leaderboard.

**Comment by jph00** [comment]
Thanks for this PR. I'm not going to merge it because fixing the issue requires a fairly thorough re-analysis of the data and rewrite of much of the prose. We'll aim to do that in a revised edition, if one gets printed.
